src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
+import React from "react";
```

Comment:
I suggest keeping your syntax consistent. try adding prettier as your default formatter in vscode. (we can look at this in our meeting.)
```suggestion
import React from 'react';
```

Reply:
Here is an example on how to test fetch call
https://jestjs.io/docs/asynchronous#asyncawait